src/IsoCodes/Bban.php

```diff
@@ -22,6 +22,10 @@ class Bban implements IsoCodeInterface
      */
     public static function validate($bban)
     {
+        if (!function_exists('bcmod')) {
```

Comment:
Instead of throwing a exception, you'd better warn our users earlier enough that `ext-bcmath`'s `bcmod` function is required for both validators. This is what Composer's `require` section was made for.